File: android/src/main/java/me/carda/awesome_notifications/utils/BooleanUtils.java
package me.carda.awesome_notifications.utils;

public class BooleanUtils {
    public static boolean getValue(Boolean booleanObject){
        return booleanObject != null && booleanObject.booleanValue();
    }
    public static boolean getValueOrDefault(Boolean booleanObject, Boolean defaultValue){
        return booleanObject == null ? getValue(defaultValue) : getValue(booleanObject);
    }
}

File: ios/Classes/lib/utils/BooleanUtils.swift
//
//  BooleanUtils.swift
//  awesome_notifications
//
//  Created by Rafael Setragni on 28/08/20.
//

import Foundation

class BooleanUtils {
    
    public static func getValue(value: Any?, defaultValue: Bool?) -> Bool {
        return value as! Bool? ?? defaultValue ?? false
    }
    
}
